### PR TITLE
feat(util): Allow more verbose logging with multiple sessions (#2985).

### PR DESCRIPTION
**Pull request by mgiambalvo**

### CODE REVIEW

**Comment by mgiambalvo** [comment]
Giving this to Craig since he'd like to do more reviews.

File: docs/referenceConf.js [hunk 1]
```diff
@@ -200,6 +200,13 @@ exports.config = {
   // Use a number less than 1 to denote unlimited. Default is unlimited.
   maxSessions: -1,
 
+  // Whether or not to buffer output when running tests on multiple browsers
+  // in parallel. By default, when running multiple browser sessions results
```
Comment (cnishina) [inline docs/referenceConf.js]: Nit: "By default, when running multiple browser sessions, the results"

File: lib/config.ts [hunk 2]
```diff
@@ -309,6 +309,15 @@ export interface Config {
      */
     maxSessions?: number;
 
+    /**
+     * Whether or not to buffer output when running tests on multiple browsers
+     * in parallel. By default, when running multiple browser sessions results
```
Comment (cnishina) [inline lib/config.ts]: Same nit.

File: lib/config.ts [hunk 3]
```diff
@@ -309,6 +309,15 @@ export interface Config {
...
+     * are buffered and not logged until the test run finishes. If true, when
+     * running multiple sessions in parallel results will be logged when each
+     * test finishes.
+     */
+    verboseMultiSessions: boolean;
```
Comment (cnishina) [inline lib/config.ts]: This is an optional interface property and should be listed as: `verboseMultiSessions?: boolean;`

**Comment by cnishina** [comment]
After these changes are made, LGTM.

**Comment by mgiambalvo** [comment]
This implements #2985

**Pull request merged**